src/main/java/sora/task/TaskList.java

```diff
@@ -0,0 +1,286 @@
+package sora.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import sora.exception.EmptyListException;
+import sora.exception.IllegalFormatException;
+import sora.exception.TaskNotFoundException;
+import sora.util.Message;
+
+/**
+ * Represents a list of tasks.
+ *
+ * @author Zhang Shi Chen
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Constructor for TaskList.
+     */
+    public TaskList() {
+        this(new ArrayList<>());
+    }
+
+    /**
+     * Overloading constructor for TaskList.
+     * Allows user to input a list of tasks.
+     *
+     * @param tasks an existing list of tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
+
+    /**
+     * Deletes the task with the same task number specified by user.
+     *
+     * @param command command entered by user (delete [task number])
```

Comment:
For all your Javadocs, may want to capitalise the first word to differentiate between parameter and description, e.g. 
```javascript
/**
* ....
* @param command Command entered by user (delete [task number])
* ....
*/
```
Thanks!

Reply:
Thx i think i miss read this one... Long way to go for modification man `checkstyle` dun seem to care about this.